xputty/widgets/xtooltip.c

```diff
@@ -49,7 +49,7 @@ Widget_t* create_tooltip(Widget_t *parent, int width, int height) {
     int x1, y1;
     Window child;
     XTranslateCoordinates( parent->app->dpy, parent->widget, DefaultRootWindow(parent->app->dpy), 0, 0, &x1, &y1, &child );
-    Widget_t *wid = create_window(parent->app, DefaultRootWindow(parent->app->dpy), x1+10, y1+10, width, height);
+    Widget_t *wid = create_window(parent->app, DefaultRootWindow(parent->app->dpy), x1+10, y1+10, width, height, true);
     Atom window_type = XInternAtom(wid->app->dpy, "_NET_WM_WINDOW_TYPE", False);
     long vale = XInternAtom(wid->app->dpy, "_NET_WM_WINDOW_TYPE_TOOLTIP", False);
```

Comment:
I don't know where you use tooltips, so I did not modify this part. likely should be the same way as the xmenu now

Reply:
tooltips will be used, for example, when a menu entry didn't fit in the menu size. Then the full entry will be shown as tooltip when hover over the entry.